tests/ast/test_pre_parser.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from vyper.ast.pre_parser import validate_version_pragma
+from vyper.exceptions import VersionException
+
+
+def test_validate_version_pragma_valid(monkeypatch):
+    _mock_version(monkeypatch)
+    validate_version_pragma(' @version 0.1.0b17', (1, 0))
+    validate_version_pragma(' @version 0.1.3', (1, 0))
+    assert True
```

Comment:
Why is this line here?

Reply:
Sorry, I just feel the need to add at least one "assert" statement in one test case. I'll remove that.